pmm-tests/pmm-framework.sh

```diff
@@ -897,6 +915,38 @@ add_clients(){
           $BASEDIR/bin/mongo --quiet --eval "printjson(db.getSisterDB('admin').runCommand({addShard: 'r${k}/localhost:${PSMDB_PORTS[$n]}'}))"
         done
 	    fi
+    elif [[ "${CLIENT_NAME}" == "pgsql" ]]; then
+      echo "Creating postgresql Dedicated User psql"
+      IP_ADDRESS=$(ip route get 8.8.8.8 | awk -F"src " 'NR==1{split($2,a," ");print a[1]}')
+      if id psql >/dev/null 2>&1; then
+        echo "yes the user psql exists"
+      else
+        echo "No, the user psql does not exist, Adding"
+        sudo adduser --disabled-password --gecos "" psql
+      fi
+      PGSQL_PORT=5431
+      for j in `seq 1  ${ADDCLIENTS_COUNT}`;do
+        PGSQL_PORT=$((PGSQL_PORT+j))
+        cd ${BASEDIR}/bin
+        if [ -d /var/lib/${NODE_NAME}_${j}/data ]; then
+          echo "PGSQL Data Directory Exist, Removing old Directory, Stopping already running Server and creating a new one"
+          sudo -H -u psql bash -c "./pg_ctl -D /var/lib/${NODE_NAME}_${j}/data -l /var/lib/${NODE_NAME}_${j}/data/logfile -o '-F -p ${PGSQL_PORT}' stop" > /dev/null 2>&1;
```

Comment:
instead of using /var/lib/ path PostgreSQL should be installed in ${BASEDIR} directory like it was implemented for other servers

Reply:
sure makes sense, I will implement those changes.